ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,106 @@
+#### Triaging of issue
```

Comment:
@bpradipt Thanks for the quick fix.  One more thing, bump this to a `#` please.  Apologies, I should have mentioned the need.  LGTM with that fix.